### PR TITLE
html movie seat booking task

### PR DESCRIPTION
**Pull request by YuliiaHordiichuk**
### html movie seat booking task

[Demo](https://yuliiahordiichuk.github.io/movie-booking/)
[Code base](https://github.com/YuliiaHordiichuk/movie-booking)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by ses3332002** [comment]
Great work!
But there are no outlines over seats in Firefox and over selected seats in Chrome when navigating with keyboard.

**Comment by ses3332002** [comment]
And one more, sorry :)
Input type="radio" (with day and time selection) are not hidden in Firefox .

**Comment by YuliiaHordiichuk** [comment]
> Great work!
> But there are no outlines over seats in Firefox and over selected seats in Chrome when navigating with keyboard.

Thanks a lot! Fixed :)

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+
+                <div class="main__info info">
+                    <div class="info__details info__details_day">
+                        <p class="info__title">Monday, 1 February</p>
+                        <ul class="info__list">
+                            <li class="info__item info__item_button">
+                                <button class="info__button">
+                                    <i class="fas fa-long-arrow-alt-left"></i>
+                                </button>
+                            </li>
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: The questionable decision put buttons into the list.

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+                            <li class="info__item">
+                                <label class="info__label">
+                                    <input type="radio" class="info__input" name="day" checked>
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Please provide input type="checkbox" with attribute value.

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+                            <li class="seats__item">c</li>
+                            <li class="seats__item">d</li>
+                            <li class="seats__item">e</li>
+                            <li class="seats__item">f</li>
+                        </ul>
+                    
+
+                        <div class="seats__template">
+                            <div class="seats__row">
+                                <input type="checkbox" class="seats__input">
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Same story as in line 81

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+                </a>
+                
+                <a href="#" class="pages-nav__btn">
+                    <i class="fas fa-long-arrow-alt-right"></i>
+                </a>
+            </nav>
+        </div>
+    </section>
+</body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Don't forget to add an empty line at the end of every file.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Remove files not for review(images).
See my comment below.

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html (hunk 2 above)
Comment (YuliiaHordiichuk) [reply submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Why should it be a checkbox and not radio type? In this case user has to choose only one option

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html (hunk 1 above)
Comment (YuliiaHordiichuk) [reply submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Changed :)

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: My bad, I mean you just need to add and fill attribute `value`.)

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+                            <li class="seats__item">c</li>
+                            <li class="seats__item">d</li>
+                            <li class="seats__item">e</li>
+                            <li class="seats__item">f</li>
+                        </ul>
+                    
+
+                        <div class="seats__template">
+                            <div class="seats__row">
+                                <input type="checkbox" class="seats__input" value="a1">
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Why you miss the attribute `name`?)

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+                            </div>
+                            <div class="seats__row">
+                                <input type="checkbox" class="seats__input" value="f11">
+                                <input type="checkbox" class="seats__input" value="f12">
+                                <input type="checkbox" class="seats__input" value="f13">
+                                <input type="checkbox" class="seats__input" value="f14">
+                            </div>
+                        </div>
+
+                        <ul class="seats__list">
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: If an order of list items matters, why you use an unordered list?

File: submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+                                </label>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="main__seats-choice seats">
+                    <p class="seats__text">Screen</p>
+                    <div class="seats__grid">
+                        <ul class="seats__list">
```
Comment (A-Ostrovnyy) [inline submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html]: Same story as in line 323.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**